### ISSUE TITLE
iOS "Add to Home Screen" Links always point to TBA homepage.

### ISSUE THREAD

**Issue by neyre** [the-blue-alliance/the-blue-alliance#2263]
If I open a page on TBA in Safari on iOS and add it to the home screen, the link on my home screen always takes me to the TBA homepage instead of whatever page I was on when I made the link

## Expected Behavior
I expect that the link that I've created on my home screen should point towards whatever page I was on when I created the link.

## Current Behavior
The link always points to the TBA homepage

## Steps to Reproduce (for bugs)
1. Open a page on Safari on iOS (like https://www.thebluealliance.com/team/254 or https://www.thebluealliance.com/event/2018hop)
2. "Add to Home Screen"
3. Click on the new link on home screen and the TBA homepage opens.
4.

## Your Environment
iOS 11.3

**Comment by bdaroz** [comment]
Some quick testing on a neighbor's iPhone shows that this is the norm for all "Add to Home Screen" functions that are done via the shortcut panel on the device. (Some pages may have a JavaScript in-webpage "add to home scree" button that can function differently.

Further all the Apple iOS examples show this landing on the site's home page.

Short version, looks like this is a "blame Apple" problem. 😦

**Comment by neyre** [comment]
@bdaroz I don't think this is the case. I just tried this on a handful of websites and all of the rest behaved as I expected with the links going to the specific page.

**Comment by bdaroz** [comment]
I stand corrected, it's a "feature": https://www.thebluealliance.com/manifest.json

(The other site I tested on has a similar file.)

**Comment by neyre** [comment]
Looks like this is caused by the 'start_url' tag in the manifest.

Not sure if this is the ideal behavior. It's a bit annoying in that if you open the app from the home screen , navigate to another page, go back to the home screen, and re-open the app, you are back at the homepage instead of the page where you left it. 

Perhaps 'start_url' can be omitted to allow users to link to specific pages?

**Comment by bdaroz** [comment]
If (when?) TBA moves to a progressive web app this may well be needed. I'll leave it to @fangeugene to jump in at this point.
